ispdb/me.com.xml

```diff
@@ -10,23 +10,15 @@
       <port>993</port>
       <socketType>SSL</socketType>
       <username>%EMAILLOCALPART%</username>

```

Comment:
```suggestion
      <username>%EMAILADDRESS%</username>
```

What about here? Does email address work as user name for this IMAP server? If so, we should change this as well.

Thanks for removing that non-working SMTP server.

Reply:
Wow, Apple's docs really ask people to use the local part for IMAP and the full email address for SMTP. So much for Apple ease of use...